frontend/packages/dev-console/integration-tests/support/step-definitions/common/operators.ts

```diff
@@ -1,12 +1,27 @@
 import { Given } from 'cypress-cucumber-preprocessor/steps';
 import { nav } from '../../../../../integration-tests-cypress/views/nav';
-import { switchPerspective } from '../../constants/global';
-import { adminNavigationMenu, perspectiveName } from '../../constants/staticText/global-text';
+import { operators, switchPerspective } from '../../constants/global';
+import { perspectiveName } from '../../constants/staticText/global-text';
 import { perspective } from '../../pages/app';
 import { operatorsPage } from '../../pages/operators-page';
+import { operatorsPO } from '@console/dev-console/integration-tests/support/pageObjects/operators-po';
+import { installOperator } from '@console/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster';
 
-Given('user has installed OpenShift Serverless Operator', () => {
```

Comment:
Why do we removed this step, Is it a duplicate one?

Reply:
Yes. Even it didn't have anything except switching the perspective